templates/4a_compensate_image_data.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "237267d0",
```

Comment:
<p><strong>"This will give us a new set of compensated images, with different values for the compensation matrix." This should maybe be at the end of the paragraph? </strong></p><p> </p><p><strong>Also specify that by default the multipliers will be applied to the noodle channel that is the main source of noise.</strong></p><p> </p><p><strong>Another point to add a note on run time that this can take upwards of 30 min to compensate 10 images X 3 multipliers. </strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>